### DISCUSSION TITLE
can this plugin be added per user account (marketplace)

### DISCUSSION THREAD

**Discussion by michelson** [pay-rails/pay#503] [Q&A] [answered]
Hello, first of all, thanks for this awesome plugin.
Could you let me know if it's possible to enable a per-user payment gateway. For use cases like marketplaces, each account has its payment account associated with selling directly. For example, a Shopify store has its payment gateway associated.
thanks

**Comment by excid3** [comment, marked as answer]
Yep, you'd need to use Stripe Connect for it. https://github.com/pay-rails/pay/blob/master/docs/marketplaces/stripe_connect.md

**Reply by michelson** [reply]
awesome, thanks
